virttest/utils_misc.py

```
@@ -4466,6 +4466,28 @@ def get_distro(session=None):
+        return distro.detect().version
+    else:
+        distro_version = ""
+        cmd = "cat /etc/os-release | grep '^VERSION_ID='"
+        try:
+            status, output = session.cmd_status_output(cmd, timeout=300)
+            if status:
+                LOG.debug("Unable to get the distro version: %s" % output)
+            else:
+                distro_version = output.split("=")[1].strip()
+        finally:
+            return distro_version
```

Ruff (0.12.2)

`return` inside `finally` blocks cause exceptions to be silenced

(B012)